app/assets/javascripts/initializers/initializeRuntimeBanner.js

```diff
@@ -56,7 +56,7 @@ function initializeRuntimeBanner() {
   // Constants - they will become dynamic (configurable by creators) in upcoming releases
   const FOREM_IOS_SCHEME = 'com.forem.app';
   const FOREM_APP_STORE_URL =
-    'https://apps.apple.com/us/app/dev-community/id1439094790';
+    'https://apps.apple.com/us/app/forem/id1536933197';
```

Comment:
I _think_ this will be our AppStore URL but we just need to double check this before merging (as soon as the app is officially published).

Reply:
Confirmed